files/scripts/supervisor-proc-exit-listener

```diff
@@ -164,9 +167,10 @@ def main(argv):
         # Check whether we need write alerting messages into syslog
         for process in process_under_alerting.keys():
             epoch_time = time.time()
-            if epoch_time - process_under_alerting[process] >= ALERTING_INTERVAL_SECS:
-                process_under_alerting[process] = epoch_time
-                generate_alerting_message(process)
+            if epoch_time - process_under_alerting[process][0] >= ALERTING_INTERVAL_SECS:
+                process_under_alerting[process][0] = epoch_time
+                process_under_alerting[process][1] += 1
+                generate_alerting_message(process, process_under_alerting[process][1])
```

Comment:
Currently `process_under_alerting[process][1]` is the number of minutes, but only because `ALERTING_INTERVAL_SECS` is set to 60. If we change `ALERTING_INTERVAL_SECS`, then this will not be a count of minutes.

Suggest either changing to seconds or add a formula to always convert to minutes, regardless of the value of `ALERTING_INTERVAL_SECS`.

Reply:
Great suggestion! Will fix.